DOCS-4245, DOCS-1870 Updates Getting Started with Private Locations and Private Locations with Podman (#16006)
DataDog/documentation: content/en/getting_started/synthetics/private_location.md

## Create your private location

> 1. You can use any Unix virtual machine with Docker or without a virtual machine. Install [Docker][4] on a machine, or use another container runtime such as [Podman][10].

Reviewer: ```suggestion
1. Install [Docker][4] on a Unix-like machine, or use another container runtime such as [Podman][10].
```
I don't understand the "or without a virtual machine" bit, so this suggestion might change the meaning of the sentence.

Reviewer 2: @hestonhoffman I think this makes sense! I feel documentation wise nothing could beat the Puppet documentation 😉

>    To get started quickly, you can install Docker on a virtual machine such as [Vagrant Ubuntu 16.04][2].

Reviewer 3: I may have read too fast, though I think the `[2]` link points to the Synthetics PL docker image and not to an Ubuntu 16.04 vagrant box. That would be `https://app.vagrantup.com/ubuntu/boxes/xenial64`

Also, the version 16.04 had its support end in April 2021, could you suggest using 22.04 (`https://app.vagrantup.com/ubuntu/boxes/jammy64`)?

    ```
    
    This command starts a Docker container and prepares your private location to run tests. Datadog recommends running the container in detached mode with proper restart policy.

>     <div class="alert alert-info">You can use another container runtime such as Podman. For more information, see the <a href="https://docs.datadoghq.com/synthetics/private_location?tab=podman#install-your-private-location">Private Locations documentation</a>.</div>

Reviewer 2: Link isn't working :(

Author: This link won't work until we merge this PR in!
